taui/src/components/edit-profile.js

```diff
@@ -561,6 +561,23 @@ class EditProfile extends PureComponent<Props> {
     return (
       <div className='account-profile__destinations'>
         <h3 className='account-profile__label'>{t('Profile.Destinations')}</h3>
+        <span>
+          <div className='account-profile__button account-profile__button--tertiary account-profile__button--iconLeft'
+            data-tip={t('Tooltips.ProfileAddresses')}
+            data-for={`tooltip-profile-addresses`}>
+            {t('Profile.WhyAddresses')}
+          </div>
+          <ReactTooltip
+            clickable
+            html
+            effect='solid'
+            place='top'
```

Comment:
```suggestion
            place='top'
            offset='{"top": -10}'
```
There's a larger gap between text and tooltip box here than the other icons, so this brings the tooltip down a little to look more standard. Not sure if there's a more elegant solution though?